upgrade/upgrade.go

```diff
@@ -152,12 +155,23 @@ func init() {
 	flag := Upgrade.Cmd.Flags()
 	flag.Bool(acl, false, "upgrade ACL from v1.2.2 to >=v20.03.0")
 	flag.Bool(dryRun, false, "dry-run the upgrade")
-	flag.StringP(alpha, "a", "127.0.0.1:9080", "Dgraph Alpha gRPC server address")
+	flag.StringP(alpha, "a", "127.0.0.1:9080",
+		"Comma separated list of Dgraph Alpha gRPC server address")
+	flag.String(slashGrpc, "", "Path to Slash GraphQL GRPC endpoint. "+
+		"If --slash_grpc_endpoint is set, all other TLS options and connection options will be "+
+		"ignored")
+	flag.String(authToken, "",
+		"The auth token passed to the server for Alter operation of the schema file. "+
+			"If used with --slash_grpc_endpoint, then this should be set to the API token issued"+
+			"by Slash GraphQL")
+	flag.String(alphaHttp, "http://127.0.0.1:8080", "Draph Alpha HTTP(S) endpoint.")
```

Comment:
doesnt this add complication for slash http endpoints? they will have different tls config i guess. does this handle that?

Reply:
I have tried it locally with TLS. I haven't tried this on slash but I have reused the same underlying function to load and attach the TLS configuration. I believe that it will work with slash as well.